backend/app/api/diagnosis.py

```
@@ -154,55 +194,66 @@ async def add_chat_message(
                 for m in db_msgs:
                     if m.role == "user":
                         chat_msgs.append(HumanMessage(content=m.content))
                     elif m.role == "assistant":
                         chat_msgs.append(AIMessage(content=m.content))
                         
-                # 답변 생성
                 chain = llm | StrOutputParser()
                 ai_reply = await chain.ainvoke(chat_msgs)
             except Exception as e:
-                print(f"Chat generation error: {e}")
+                logger.error(f"Chat generation error for history {history_id}: {e}", exc_info=True)
```

Ruff (0.15.10)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)